scripts/agent/build-jail.sh

```diff
@@ -0,0 +1,26 @@
+#! /bin/bash
```

Comment:
Jail already ships its own [binary distribution](https://github.com/lablup/backend.ai-jail/releases) so this script is redundant.